docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-hints-autodiscover.asciidoc

```diff
@@ -92,7 +92,7 @@ https://github.com/elastic/elastic-agent/tree/main/deploy/kubernetes/elastic-age
 [discrete]
 == Configure hints autodiscovery
 
-To enable hints, you must add `hints.enabled: true` to the provider's configuration:
+To enable hints, you must add `hints.enabled: true` to the provider's configuration of https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml[{agent}-standalone manifest] :
```

Comment:
```suggestion
To enable hints, you must add `hints.enabled: true` to the provider's configuration:
```
I don't see a reason pointing to a specific manifest here since the way that users can deploy Agent can be trough multiple channels (eck, helm etc).
But if we think we need to add this one you will need to fix the link to not point to the `main` branch but to the respective release's one. The reason for this is that if we update `main` for example then all the previous docs releases will be pointing to main branch. This will be inconsistent.